File: code/a4204.pas
program Bai204;
  var n, a: longint;
  function SNT(x: longint) : boolean;
    var i :longint;
    begin
    if x < 2 then exit(false);
    for i := 2 to x div 2 do 
        if x mod i = 0 then exit(false);
        exit(true);
    end;
begin
  readln(n);
  n := n + 1;
  while SNT(n) = false do n := n + 1;
      writeln (n);
end.

File: code/a4204.cpp
#include <iostream>
using namespace std;

bool snt(int n){
    int x;
    cin>>n;
    if (n<2)
        return false;
    for (x=2; x<=n-1; x++)
        if (n%x==0)
            return false;
    return true;
}

int main() 
{
    int n;
    cin>>n;
    n++;
    while (snt(n)==false)
        n++;
    cout<<n;
    return 0;
}